pkgs/development/libraries/mesa/default.nix

```diff
@@ -28,9 +28,9 @@
       "tegra" # Nvidia Tegra SoCs
       "v3d" # Broadcom VC5 (Raspberry Pi 4)
       "vc4" # Broadcom VC4 (Raspberry Pi 0-3)
-    ] ++ lib.optionals stdenv.isx86_64 [
-      "iris" # new Intel, could work on non-x86_64 with PCIe cards, but doesn't build as of 22.3.4
-      "crocus" # Intel legacy, x86_64 only
+    ] ++ lib.optionals stdenv.isx86 [
```

Comment:
Fails eval. `stdenv.hostPlatform.isx86` perhaps?

Reply:
Maybe `stdenv.isx86_32`?

Reply:
`stdenv.hostPlatform.isx86` should be easiest, as we want both bit-ness.